src/api.tsx

```diff
@@ -406,16 +407,17 @@ export async function deleteEvent(id: string, deleteModifiedInstance: boolean =
     return [];
 }
 
-export async function updateMediaInfo(imageInfo: ImageInfo) {
+export async function updateMediaInfo(imageInfo: MediaResource) {
     if (imageInfo._ref) {
+        const keywords = imageInfo.keywords?.toString().split(",");
```

Comment:
how does the user know that keywords are separated by commas?
I would expect some text saying this, or maybe also be tolerant to spaces?
also, maybe trim spaces of the keywords?

Reply:
fixed

Reply:
added trim and change the delimiter to space